README.md

```diff
@@ -106,7 +106,7 @@ $ npm run migration:revert
 $ docker build -t my-app .
 
 # run container from image
-$ docker run -p 3000:3000 --volume `pwd`:/usr/src/app --env-file .env my-app
+$ docker run -p 3000:3000 --volume 'pwd':/usr/src/app --env-file .env my-app
```

Comment:
```suggestion
$ docker run -p 3000:3000 --volume 'pwd':/usr/src/app --network host --env-file .env my-app
```
@Saad-Amjad I also faced issue locally and had to also add this `--network` option.
Can you let me know if this one is better or not. :bow: